src/pages/Spreadsheet.tsx

```diff
@@ -63,6 +64,18 @@ export const Spreadsheet = (): React.ReactElement => {
       </p>
 
       <p>{spreadsheet('Use the spreadsheet here')}</p>
+
+      <Alert variant="info">
+        <strong>Spreadsheet version 2.0 coming soon:</strong> We are looking for
+        beta testers for a new version of the spreadsheet. If you are interested
+        in using the microCOVID spreadsheet with your household/pod to track
+        your collective COVID risk and are interested in giving us feedback on
```

Comment:
Weird to use "interested" twice in the same sentence; maybe "willing to give us feedback on..."

Reply:
good catch. thanks.